mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -224,6 +224,8 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_32326 getPreferredEquipmentSlot (Lnet/minecraft/class_1799;)Lnet/minecraft/class_1304;
 		ARG 0 stack
 	METHOD method_33190 canTakeDamage ()Z
+	METHOD method_33579 updateValuesFromSpawnPacket (Lnet/minecraft/class_2610;)V
+		ARG 1 mobSpawnS2CPacket
```

Comment:
```suggestion
		ARG 1 packet
```
Should the method be just called `loadFromPacket` (if it loads most of the entity attributes)? Or is the packet just a small section of entity information?

Reply:
`loadFromPacket` looks better since it updates the significant part of the entity including uuid and entityId.